src/main/app/case/definition.ts

```diff
@@ -2420,3 +2420,25 @@ export interface C1ASafteyConcerns {
       statement: string;
     };
   };
+
+  export type ChildrenDetailsTranslations = {
+    issueOrderList?: any;
```

Comment:
@giorgi-hmcts Please check these are really required ?

Reply:
Not required. Removed them as well.